src/Symfony/Component/Security/Http/Tests/RememberMe/PersistentRememberMeHandlerTest.php

```diff
@@ -118,7 +118,7 @@ public function testConsumeRememberMeCookieExpired()
         $this->tokenProvider->expects($this->any())
             ->method('loadTokenBySeries')
             ->with('series1')
-            ->willReturn(new PersistentToken(InMemoryUser::class, 'wouter', 'series1', 'tokenvalue', new \DateTime('-'.(31536000 - 1).' years')));
+            ->willReturn(new PersistentToken(InMemoryUser::class, 'wouter', 'series1', 'tokenvalue', new \DateTime('-'.(31536000 - 1).' months')));
```

Comment:
/cc @wouterj WDYT?

Reply:
To me, seems like we can even make this `'-'.(31536000 +1).' seconds'`?